WearBuildConfig/mobile/build.gradle

```diff
@@ -0,0 +1,71 @@
+apply plugin: 'com.android.application'
+apply from: "$rootProject.projectDir/WearBuildConfig/shared-config/android-signing.gradle"
```

Comment:
this could be a more idiomatic `rootProject.file('WearBuildConfig/shared-config/android-signing.gradle')`